### PR TITLE
Preparing for release

### PR DESCRIPTION
**Pull request by kylec32**

### CODE REVIEW

**Comment by casz** [comment]
CI does not like a repo without `.gitignore` :) https://ci.jenkins.io/blue/rest/organizations/jenkins/pipelines/Plugins/pipelines/bitbucket-approval-filter-plugin/branches/depend-on-release/runs/3/nodes/8/steps/58/log/?start=0

File: .mvn/maven.config [hunk 1]
```diff
@@ -1,2 +1,4 @@
 -Pconsume-incrementals
 -Pmight-produce-incrementals
+-Pconsume-incrementals
```
Comment (casz) [inline .mvn/maven.config]: duplicated :)

**Comment by kylec32** [comment]
@casz I'm all ears on improvement ideas you have. (Although I do think it has ben released now even though CI hated the code).

**Comment by casz** [comment]
my improvements are in #5 :dancer:

**Comment by casz** [comment]
@kylec32 your welcome to cherry pick my changes :+1:

File: .mvn/maven.config (hunk 1 above)
Comment (kylec32) [reply .mvn/maven.config]: Crazy magic tool duplicating my stuff. :-) Good catch.

**Comment by casz** [comment]
You want to avoid merges of remote branches into local. It would be easier to reset your local branch to the remote, instead of merging on your local branch :smile:

**Comment by kylec32** [comment]
I for sure did all that merging more messily than I needed to. Looks like everything got published and I was able to install from Jenkins. 🎉 

Thank you for all of your help @casz! In both this plugin and bitbucket-branch-source plugin. You have been patient and super helpful with everything! 👍

**Pull request merged**